users/models.py

```diff
@@ -65,7 +78,7 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     USERNAME_FIELD = 'email'
     EMAIL_FIELD = 'email'
-    REQUIRED_FIELDS = ['first_name', 'last_name', 'birth_date']
+    REQUIRED_FIELDS = ['first_name', 'last_name', 'birth_date', 'user_type']
```

Comment:
@Micha546 This is in class 'User', aren't these required fields for ANY user? in such case 'user_type' should be required.

Reply:
REQUIRED_FIELDS is an array of all the fields that will be prompted when you call `creatsuperuser`, in our case we call
`pipenv run python manage.py createsuperuser`. if the field has a default, there's no need to ask the admin to choose owner or seeker, because he's "both" anyways

Reply:
I get it now, I'll remove this from the list.